visualisation/backend/utils.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef SERENITY_VISUALISATION_UTILS_HPP
+#define SERENITY_VISUALISATION_UTILS_HPP
+
+#include <iostream>
+#include <iomanip>
+#include <string>
+
+#include "glog/logging.h"
+
+#include "stout/try.hpp"
+
+namespace mesos {
+namespace serenity {
+
+class Precision {
+ public:
+  static constexpr uint32_t MILI = 3;
+  static constexpr uint32_t MIKRO = 6;
+  static constexpr uint32_t NANO = 9;
+};
+
+inline std::string DblTimestampToString(
+    const double_t _timestamp,
+    const uint32_t _precision = Precision::MILI) {
+  std::stringstream timeStream;
+  timeStream << std::fixed << std::setprecision(_precision) << _timestamp;
+
+  std::string timeStr(timeStream.str());
+  timeStr.erase(std::remove(timeStr.begin(), timeStr.end(), '.'),
+                timeStr.end());
+
+  return timeStr;
+}
+
+inline Try<std::string> GetHostName() {
```

Comment:
How about getting this from the slave instead (hitting the slave endpoint)? We will probably need that information anyway.